### PR TITLE
Update quickstart example notebook

### PR DESCRIPTION
**Pull request by alex-zenml**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Fixed incorrect arguments being passed into fashion MNIST pipeline definition.
Fixed incorrect index value being passed in for post-execution workflow.

### CODE REVIEW

File: examples/quickstart/quickstart.ipynb [hunk 1]
```diff
@@ -550,8 +550,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "mnist_eval_step = mnist_run.steps[3]\n",
-    "fashion_mnist_eval_step = fashion_mnist_run.steps[3]"
+    "mnist_eval_step = mnist_run.steps[2]\n",
```
Comment (htahir1) [inline examples/quickstart/quickstart.ipynb:553]: We can replace this with the `get_step(name='evaluator')` paradigm to make it super clear.

File: examples/quickstart/quickstart.ipynb [hunk 2]
```diff
@@ -407,7 +407,7 @@
     "# Initialise a new pipeline\n",
     "second_pipeline = mnist_pipeline(\n",
     "    importer=importer_fashion_mnist(),\n",
-    "    trainer=trainer(config=TrainerConfig(epochs=1)),\n",
+    "    trainer=trainer(),\n",
```
Comment (htahir1) [inline examples/quickstart/quickstart.ipynb:410]: Isn't it cooler to add the config to show it off?

**Review by htahir1 — CHANGES_REQUESTED** [review]
Small edits

File: examples/quickstart/quickstart.ipynb (hunk 2 above)
Comment (alex-zenml) [reply examples/quickstart/quickstart.ipynb:410]: I get the following error when I run that:

```
---------------------------------------------------------------------------
StepInterfaceError                        Traceback (most recent call last)
<ipython-input-23-8e51e730bb32> in <module>()
      2 first_pipeline = mnist_pipeline(
      3     importer=importer(),
----> 4     trainer=trainer(config=TrainerConfig(epochs=1)),
      5     evaluator=evaluator(),
      6 )

/usr/local/lib/python3.7/dist-packages/zenml/steps/base_step.py in __init__(self, *args, **kwargs)
    164         if len(kwargs) > maximum_kwarg_count:
    165             raise StepInterfaceError(
--> 166                 f"Too many keyword arguments ({len(kwargs)}, "
    167                 f"expected: {maximum_kwarg_count}) passed when "
    168                 f"creating a '{self.step_name}' step."

StepInterfaceError: Too many keyword arguments (1, expected: 0) passed when creating a 'trainer' step.
```

I didn't dig into exactly what this was connected to, so thought I'd just remove for now. Is there a quick fix that you can see?

**Pull request merged**